### PR TITLE
Clipping planes support cleanup

### PR DESCRIPTION
**Pull request by ggetz**
Addresses @lilleyse 's comments in https://github.com/AnalyticalGraphicsInc/cesium/pull/6079

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@ggetz, thanks for the pull request! Maintainers, we have a signed CLA from @ggetz, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/Model.js [hunk 1]
```diff
@@ -1135,6 +1137,15 @@ define([
...
+     * @param {Color} [options.color=Color.WHITE] A color that blends with the model's rendered color.
+     * @param {ColorBlendMode} [options.colorBlendMode=ColorBlendMode.HIGHLIGHT] Defines how the color blends with the model.
+     * @param {Number} [options.colorBlendAmount=0.5] Value used to determine the color strength when the <code>colorBlendMode</code> is <code>MIX</code>. A value of 0.0 results in the model's rendered color while a value of 1.0 results in a solid color, with any value in-between resulting in a mix of the two.
+     * @param {Color} [options.silhouetteColor=Color.RED] The silhouette color. If more than 256 models have silhouettes enabled, there is a small chance that overlapping models will have minor artifacts.
+     * @param {Number} [options.silhouetteSize=0.0] The size of the silhouette in pixels.
+     * @param {ClippingPlaneCollection} [options.clippingPlanes] The {@link ClippingPlaneCollection} used to selectively disable rendering the model. Clipping planes are not currently supported in Internet Explorer.
```
Comment (lilleyse) [inline Source/Scene/Model.js]: Thanks... I didn't realize so many options were missing.

File: Source/Core/ClippingPlaneCollection.js [hunk 2]
```diff
@@ -362,6 +364,15 @@ define([
         return intersection;
     };
 
+    /**
+     * Determines if rendering with clipping planes is supported.
+     *
+     * @returns {Boolean} <code>true</code> if ClippingPlaneCollections are supported; otherwise, returns <code>false</code>
+     */
+    ClippingPlaneCollection.isSupported = function() {
```
Comment (lilleyse) [inline Source/Core/ClippingPlaneCollection.js]: Does this warrant an update to CHANGES.md, or too minor? @pjcozzi?

Comment (mramato) [reply Source/Core/ClippingPlaneCollection.js]: Anything that changes the public API always warrants a CHANGEs.md update.

**Comment by ggetz** [comment]
Thanks @lilleyse, updated CHANGES.md.

**Comment by lilleyse** [comment]
Looks good.

**Pull request merged**